.gitignore

```diff
@@ -12,3 +12,23 @@ config.status
 Makefile.config
 HYPRE_config.h
 autom4te.cache
+src/TAGS
+hypre/
+
+###############
+# Executables
+###############
+ams_driver
```

Comment:
I think we should preface these with `src/test/` and `src/examples`?

Reply:
That makes sense! Fixed with [9d010e3](https://github.com/hypre-space/hypre/pull/599/commits/9d010e392c8e5cd4b00d3e5ad140d83b743a6841)